libraries/platforms/dromajo-vcs/software/include/bp_utils.h

```diff
@@ -0,0 +1,113 @@
+// BlackParrot utilities header file
+// Note: This file is included here instead of the existing code
+// in the perch library in order to allow for easy changes to the finish code
+// in the future
+
+#ifndef BP_UTILS_H
+#define BP_UTILS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdint.h>
+#include <bsg_newlib_intf.h>
+#include <bsg_manycore.h>
+
+// Memory mapped addresses to interact with the host
+#define MC_ARGS_START_EPA_ADDR 0x0000
+#define MC_ARGS_FINISH_EPA_ADDR 0x00FF
+#define MC_CONFIG_START_EPA_ADDR 0x0100
+#define MC_CONFIG_FINISH_EPA_ADDR 0x01FF
+#define MC_RESET_DONE_EPA_ADDR 0x0200
+#define MC_TX_VACANT_EPA_ADDR 0x0300
+#define MC_FINISH_EPA_ADDR 0xEAD0
+#define MC_TIME_EPA_ADDR 0xEAD4
+#define MC_FAIL_EPA_ADDR 0xEAD8
+#define MC_STDOUT_EPA_ADDR 0xEADC
+#define MC_STDERR_EPA_ADDR 0xEAE0
+#define MC_BRANCH_TRACE_EPA_ADDR 0xEAE4
+#define MC_PRINT_STAT_EPA_ADDR 0xEA0C
```

Comment:
Are these the same addresses as I commented on above?

Reply:
Yes, most of them are the same except ones I added to do different things like retrieve arguments, retrieve info from the config ROM and check if reset is complete. The others are the same.

Reply:
This is all now a part of `bp_hb_platform.h`. So no more duplication. Both RISC-V (Dromajo/BlackParrot) and x86 (BlackParrot Host; Simulator) use the same header.